docs/v0.7/en/user-guide/clients/http-api.md

```diff
@@ -49,11 +49,55 @@ http://localhost:4000/v1/sql?db=public
 InfluxDB uses its own authentication format, see [InfluxDB](./influxdb-line.md) for details.
 :::
 
-## Write Data
+## Time zone
+
+GreptimeDB supports the `X-Greptime-Timezone` field in the header of the HTTP protocol.
```

Comment:
```suggestion
GreptimeDB supports the `X-Greptime-Timezone` header in HTTP requests.
```

IIRC HTTP Xxx header is a formal description. cc @nicecui  - https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type "a Content-Type header"